### ISSUE TITLE
Parameter "maxIterations" is ignored in Louvain Community Detection algo

### ISSUE THREAD

**Issue by ThachNgocTran** [neo4j/graph-data-science#81]
## Expected Behavior
[Documentation](https://neo4j.com/docs/graph-data-science/current/algorithms/louvain/) on Louvain algorithm states:

maxIterations | Integer | 10 | yes | The maximum number of iterations that the modularity optimization will run for each level.
-- | -- | -- | -- | --

... which means the **users can modify the number** of max iterations, despite it being optional (and having default value of `10`).

## Actual Behavior

Latest version of branch 1.3 has this parameter [hardcoded](https://github.com/neo4j/graph-data-science/blob/f90df69a1f80b8a126b54429ab558f49daeb5f37/algo/src/main/java/org/neo4j/graphalgo/louvain/Louvain.java#L152).

```
ModularityOptimizationStreamConfig modularityOptimizationConfig = ImmutableModularityOptimizationStreamConfig
            .builder()
            .maxIterations(10)
```

### Versions
+ Graph Data Science library, branch 1.3 (checked on 02.10.2020)

**Comment by ThachNgocTran** [comment]
Shall I create a Pull Request?

**Comment by Mats-SX** [comment]
@ThachNgocTran Thanks for finding this! We will pick this up soon and fix it in the next `1.3` patch release. If you would like to issue a pull-request, please also make sure to sign the [contribution agreement](https://github.com/neo4j/graph-data-science/blob/master/CONTRIBUTING.md), and we'll make sure to take your changes into account in the fix. We will take care of this issue even if you do not submit a PR though so as you please :)

**Comment by ThachNgocTran** [comment]
Hi @Mats-SX 

Thanks for the response. I created the Pull Request here: https://github.com/neo4j/graph-data-science/pull/83

I also read and signed our CLA.
